tests/tests/test_host_kdump.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/bash
+#
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+SSH_OPTIONS=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
```

Comment:
Doesn't this need to get passed to each of the `ssh` commands below (`ssh $SSH_OPTIONS ubuntu@$DEVICE_IP ...`)? I don't think it's a standard shell var that `ssh` reads, is it?

Reply:
dangit, i'll slow down :-)

Reply:
https://github.com/canonical/tdx/pull/237/commits/09beade5530cad73a37dc7ebe03bf71647f289a0